stable/mlrun-kit/templates/jupyter-notebook/pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.jupyterPersistencyStandalone.pv.create -}}
```

Comment:
`jupyterNotebook.persistency.pvc.create`

Reply:
pv,pvc in separate block in order not to case brake compatibility in original mlrun values.yaml